doc/contributor/cexts.md

```diff
@@ -14,7 +14,7 @@ Clone Sulong as a sibling directory and build it:
 # In the truffleruby repo
 git clone https://github.com/graalvm/sulong.git ../sulong
 cd ../sulong
-mx build
+jt mx build
```

Comment:
I think it would be nice if we could just keep it as `jt build` for Sulong as well. Otherwise, you'll have to remember to do `jt build` for TruffleRuby and `jt mx build` for Sulong. But, if that makes things too complicated for jt.rb, this is fine as well.